base/src/main/java/com/smartdevicelink/managers/lifecycle/BaseSystemCapabilityManager.java

```diff
@@ -246,7 +255,11 @@ public WindowCapability getWindowCapability(int windowID) {
         for (WindowCapability windowCapability : display.getWindowCapabilities()) {
             int currentWindowID = windowCapability.getWindowID() != null ? windowCapability.getWindowID() : PredefinedWindows.DEFAULT_WINDOW.getValue();
             if (currentWindowID == windowID) {
-                return windowCapability;
+                // Clone WindowCapability to prevent modification of stored WindowCapability in SystemCapabilityManager
+                WindowCapability updatedCaps = (WindowCapability) windowCapability.clone();
+                // A null windowID is assumed to be the DefaultWindow according to the spec, but that can be hard for developers to check, so set it explicitly.
+                updatedCaps.setWindowID(windowID);
+                return updatedCaps;
```

Comment:
```suggestion
                WindowCapability windowCapabilityCopy = (WindowCapability) windowCapability.clone();
                // A null windowID is assumed to be the DefaultWindow according to the spec, but that can be hard for developers to check, so set it explicitly.
                windowCapabilityCopy.setWindowID(windowID);
                return windowCapabilityCopy;
```
Original variable name was really just a placeholder, so perhaps this would be better this time around? (hard to come up with a good name for this one)